Gemfile

```diff
@@ -15,14 +15,15 @@ gem 'image_processing', '1.9.3'
 gem 'mini_magick', '4.9.5'
 gem 'active_storage_validations', '0.8.2'
 gem "aws-sdk-s3", require: false
+gem 'pry-rails'
```

Comment:
This should probably exist in the dev/test group below

Reply:
true dat